client/src/components/ExerciseList/ExerciseListContainer.jsx

```diff
@@ -33,3 +33,19 @@ class ExerciseListContainer extends React.Component {
 }
 
 export default ExerciseListContainer;
+
+import { connect } from 'react-redux';
+import ExerciseEventList from './ExerciseEventList';
+
+const mapStateToProps = state => {
+  const user_id = state;
```

Comment:
see the shape of our state in `src/store/initialState.js`

Reply:
Does user store the user's id or the user as an object?